### PR TITLE
thumb clicks work

### PR DESCRIPTION
**Pull request by DanBuda11**
Photo/title thumbs on stories page take the user to the details page; updated title tag to h2 for Dana's styling.

### CODE REVIEW

File: public/scripts/components/pages/Stories.js [hunk 1]
```diff
@@ -1,11 +1,65 @@
...
+				
 			</section>
 		);
+			
 	}
-});
\ No newline at end of file
+
+});
+export default StoryList;
```
Comment (alarner) [inline public/scripts/components/pages/Stories.js]: The Stories component probably shouldn't be modified in this PR. Any idea why this file is changed?

**Comment by willcook4code** [comment]
👍

File: public/scripts/components/sub-components/PicTitleThumb.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import {Link} from 'react-router';
 
 export default React.createClass({
 	render: function() {
 		return (
+<<<<<<< HEAD
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/PicTitleThumb.js]: pls remove merge syntax

**Comment by alarner** [comment]
👍

**Comment by NicerHugs** [comment]
As soon as Martin's PR is merged in these links will not be pointing to the proper url. The route to the details page will have changed to include the storyId

File: public/scripts/components/sub-components/PicTitleThumb.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ export default React.createClass({
 	render: function() {
 		return (
 		<div className="story-thumb-box" >
-			<Link to="/details">
+			<Link to="/stories/:storyId/details">
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/PicTitleThumb.js]: in this case you will be linking to the LITERAL string `'/stories/:storyId/details'`.  You probably want to link to something like `'/stories/3/details'`, for example.

**Comment by alarner** [comment]
@DanBuda11 You need to actually replace :storyId with the actual story id

**Comment by tinydinosaurs** [comment]
👍 🎓 🍰 🎯 👍

**Comment by NJHawthorne** [comment]
👍

**Comment by alarner** [comment]
👍

**Pull request merged**